wit/wasi-filesystem.wit.md

```diff
@@ -349,6 +349,19 @@ enum advice {
 type descriptor = u32
 ```
 
+## `get-preopens`
+```wit
+/// Get preopened file descriptors.
+///
+/// Provided by the environment as a pair of a path name and a descriptor
+/// of a directory.
+///
+/// Morally, these are a value import, but until value imports are available
+/// in the component model, this import function should be called at most
+/// once, and subsequent calls should trap.
+get-preopens: func() -> list<tuple<descriptor, string>>
```

Comment:
Could you add some documentation mentioning that this creates new handles each time it's called, similar to what eg. [default-monotonic-clock](https://github.com/WebAssembly/wasi-clocks/blob/main/wit/wasi-default-clocks.wit.md#default-monotonic-clock) has?

Reply:
I think that by emulating a value import we don't want to create a new handle each call, though - we want to give ownership of an existing resource to the caller. Either way, we'll have to work out these semantics for both this function, and for clocks, once we actually have resources, so I think we should save it for then